buildpack.toml

```diff
@@ -3,7 +3,7 @@ api = "0.2"
 [buildpack]
 id = "salesforce/nodejs-fn"
 name = "Salesforce NodeJS Function Structured Middleware Buildpack"
-version = "1.9.7"
+version = "2.0.0"
```

Comment:
Yeah I was going to ask about this - seems like it's worth bumping major version.